README-dev.md

```diff
@@ -136,4 +136,16 @@ common cause of this is if you are running the server in both production
 mode and developer mode on the same host. Simply remove the file or change
 ownership permissions so that you can read/write it.
 
-
+**Q**: I get ```  File "/usr/local/lib/python2.7/site-packages/bs4/builder/_html5lib.py", line 57, in <module>
+    class TreeBuilderForHtml5lib(html5lib.treebuilders._base.TreeBuilder):
+AttributeError: 'module' object has no attribute ‘_base’``` or missing python
+ package when I run server in PyCharm.
+
+**A:** From Tim, run these steps:
+1. Close PyCharm
+2. `make kill` / `docker-compose -p projecta kill`
+3. `make rm` / `docker-compose -p projecta rm`
+4. `make build`
+5. `make build-devweb`
+6. `make devweb`
+7. start PyCharm
```

Comment:
Can you edit the main workflow? 
https://github.com/ismailsunni/projecta/blob/3ecf4faf7975766746b05aa963b077d862c8bfba/README-docker.md#production

Maybe add a new block which commands to do when the code is updated?

When I have an error, I don't think going to the FAQ. It's more like a bug for me and not a question. And I never been to this page.

make dbrestore, make migrate are missing from this list.